network-api/networkapi/buyersguide/tests.py

```diff
@@ -440,18 +460,455 @@ def name_change_changes_slug(self):
+        product_page = ProductPage.objects.first()
+        if not product_page:
+            product_page = ProductPage(
+                slug='product-page',
+                slug_en='product-page',
+                title='Product Page',
+                title_en='Product Page',
+                live=True,
+            )
+            self.bg.add_child(instance=product_page)
+            product_page.save_revision().publish()
+        return product_page
 
 
 @override_settings(STATICFILES_STORAGE="django.contrib.staticfiles.storage.StaticFilesStorage")
-class AboutViewTest(TestCase):
-    def test_localised_about(self):
-        url = '/fr/privacynotincluded/about/'
+class TestBuyersGuidePage(BuyersGuideTestMixin):
+
+    def test_buyersguide_url(self):
+        self.assertEqual(self.bg.slug, 'privacynotincluded-new')
+        response = self.client.get(self.bg.url)
+        self.assertEqual(response.status_code, 200)
+
+    def test_buyersguide_about_how_to_use_view(self):
+        url = self.bg.reverse_subpage('how-to-use-view')
+        self.assertEqual(url, 'about/')
+        response = self.client.get(self.bg.url + url)
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'about/how_to_use.html')
+
+    def test_buyersguide_about_why_view(self):
+        url = self.bg.reverse_subpage('about-why-view')
+        self.assertEqual(url, 'about/why/')
+        response = self.client.get(self.bg.url + url)
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'about/why_we_made.html')
+
+    def test_buyersguide_about_press_view(self):
+        url = self.bg.reverse_subpage('press-view')
+        self.assertEqual(url, 'about/press/')
+        response = self.client.get(self.bg.url + url)
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'about/press.html')
+
+    def test_buyersguide_about_contact_view(self):
+        url = self.bg.reverse_subpage('contact-view')
+        self.assertEqual(url, 'about/contact/')
+        response = self.client.get(self.bg.url + url)
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'about/contact.html')
+
+    def test_buyersguide_about_methodology_view(self):
+        url = self.bg.reverse_subpage('methodology-view')
+        self.assertEqual(url, 'about/methodology/')
+        response = self.client.get(self.bg.url + url)
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'about/methodology.html')
+
+    def test_buyersguide_about_mss_view(self):
+        url = self.bg.reverse_subpage('min-security-view')
+        self.assertEqual(url, 'about/meets-minimum-security-standards/')
+        response = self.client.get(self.bg.url + url)
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'about/minimum_security.html')
```

Comment:
These are effectively all the same test, for different URLs: let's make this its own code, so that all these tests are one liners:
```python
def about_url_test(self, view_name, target_url, template):
    url = self.bg.reverse_subpage(view_name)
    self.assertEqual(url, target_url)
    response = self.client.get(self.bg.url + url)
    self.assertEqual(response.status_code, 200)
    self.assertTemplateUsed(response, f'about/{template}.html)
```

Turning this code into:
```python
def test_buyersguide_about_how_to_use_view(self):
    self.about_url_test(`how-to-use-view`, `about/`, `how_to_use`)

...

def test_buyersguide_about_mss_view(self):
    self.about_url_test('min-security-view', 'about/meets-minimum-security-standards/', 'minimum_security')
```